src/lib/zAuction.ts

```diff
@@ -57,29 +48,18 @@ interface BidPostInterface {
 	signedMessage: string;
 }
 
-interface BidAcceptInterface {
-	signedMessage: string;
-	auctionId: string;
-	account: string;
-	bidAmount: string;
-	contractAddress: string;
-	tokenId: string;
-	minimumBid: string;
-	startBlock: string;
-	expireBlock: string;
-}
-
-function getApiEndpoint() {
-	// const chainSelector = useChainSelector();
-	let chain = 42; //useChainSelector().selectedChain;
-	switch (chain) {
-		case 1:
-			return 'https://zproduction.ilios.dev/api';
-		case 42:
-			return 'https://zproxy.ilios.dev/api';
-		default:
-			return 'Unsupported Chain';
-	}
+function setApiEndpoints(apiName: string) {
```

Comment:
`apiName` is a misleading parameter name. You are passing in an URI here, not the name of an API.

Therefore a name like "baseUri` or `baseApiUri` or `BaseApiEndpoint` would be more fitting.